### PR TITLE
Adding open font Selawik for Text Prefab

### PR DESCRIPTION
**Pull request by cre8ivepark**
Per discussion in the issue #377 and #425 , added open font [Selawik ](https://github.com/Microsoft/Selawik)and updated text prefabs, materials and test scenes. Now Text Prefabs and Samples have Selawik font assigned as default. Developers can follow [existing instructions from this readme file ](https://github.com/Microsoft/HoloToolkit-Unity/blob/master/Assets/HoloToolkit/UI/README.md)to apply other fonts.

selawikfonts

textprefabupdate_selawik

### CODE REVIEW

File: Assets/HoloToolkit/UI/Fonts/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,91 @@
+Copyright 2015, Microsoft Corporation (www.microsoft.com), with Reserved Font Name Selawik.  All Rights Reserved.  Selawik is a trademark of Microsoft Corporation in the United States and/or other countries.
```
Comment (NeerajW) [inline Assets/HoloToolkit/UI/Fonts/LICENSE.txt]: TODO: Need to find out if having a SIL license if ok under an MIT project.

File: Assets/HoloToolkit/UI/README.md [hunk 2]
```diff
@@ -26,7 +26,8 @@ Common useful UI prefabs
 
 UI Text Mesh prefab with optimized scaling factor at 2-meter distance. (Please read the instructions below)
 
-**IMPORTANT**: To use Text prefabs, please make sure to import SegoeUI.TTF font file from your system font folder (C:\Windows\Fonts\)  
+**IMPORTANT: Text Prefab uses oepn source font 'Selawik'. To use Text Prefab with different font, please import font file and follow the instruction below. Below example shows how to use 'Segoe UI' font with Text Prefab.**
```
Comment (NeerajW) [inline Assets/HoloToolkit/UI/README.md]: Typo: open -> open

File: Assets/HoloToolkit/UI/Fonts/LICENSE.txt (hunk 1 above)
Comment (HodgsonSDAS) [inline Assets/HoloToolkit/UI/Fonts/LICENSE.txt]: Any word on this?

Comment (NeerajW) [inline Assets/HoloToolkit/UI/Fonts/LICENSE.txt]: Yes we got approval from LCA on this! WOOT!

**Pull request merged**